src/main/resources/library/attribute-definitions.json

```diff
@@ -325,13 +325,6 @@
       "indexable": false,
       "consentCode": "RS-PD"
     },
-    "library:RS-POP": {
```

Comment:
Why is this going away?

Reply:
Follow-up: I got an answer. It wasn't really being used so we want to clean it up now rather than expose it in a public API.